gdal/doc/source/development/rfc/rfc83_use_of_project_sponsorship.rst

```diff
@@ -0,0 +1,228 @@
+includes PROJ, libtiff, libgeotiff, shapelib, GEOS, openjpeg. If during the
+investigation of an issue, a co-maintainer finds an issue in another open source
+component and can come up with a patch, they are also welcome to do it.
+
+A typical share of time could be 75% for GDAL and 25% for the other projects.
+
+GDAL is a vast project. It is not expected that each co-maintainer has expertise
+in all its aspects. Applicants will mention their domains of interest, which can
+evolve over time, and coordinate with other stakeholders in how tasks are
+assigned.
+
+Direction
++++++++++
+
+.. Mostly taken from RFC9, but with a key difference on the priority given to
+   issues coming from sponsors
+
+The co-maintainer is generally subject to the project PSC. However, for day to
+day decisions one PSC member might be designated as the supervisor for the
+co-maintainer (PSC members that are funded through this program will use existing
+decision making processes to prioritize their work). This supervisor will
+coordinate with the co-maintainer via email or IRC discussions.
+
+.. I was unsure how to handle that situation, but the supervisor role might be
+   somewhat time consuming, and if no PSC member is funded to supervise a funded
+   PSC member that might not work well in practice.
+
+The supervisor will try to keep the following in mind when prioritizing tasks:
+
+- Tickets that are regressions compared to a previous release are of higher
+  priority than other tasks
+- Bug reports should be given higher priority than other tasks. (bug reports
+  from sponsors will *not* qualify as such for higher priority treatment)
+- Areas of focus identified by the PSC should be given higher priority than
+  other tasks.
+- Bugs or needs that affect many users should have higher priority.
+- The co-maintainer should be used to take care of work that no one else is
+  willing and able to do (i.e. fill the holes, rather than displacing volunteers)
+- Try to avoid tying up the co-maintainer on one big task for many weeks unless
+  directed by the PSC.
+- The co-maintainer should not be directed to do work for which someone else is
+  getting paid.
+
+Substantial new development projects will only be taken on by the co-maintainer
+with the direction of a PSC motion (or possibly an RFC designating the co-maintainer
+to work on a change).
+
+Note that the co-maintainer and the co-maintainer supervisor are subject to the
+normal RFC process for any substantial change to GDAL.
+
+Reporting
++++++++++
+
+The co-maintainer will produce a monthly report (typically in a shared spreadsheet)
+with the tasks he/she has tackled, typically pointing at tickets and pull requests,
+and the amount of hours spent during the reported period.
+
+Who can apply ?
++++++++++++++++
+
+Applicants should be individuals, either self-employed or employed by a company
+agreeing to allocate some of their time to such activities, that have been a
```

Comment:
I'm sorry for, perhaps, sticking a cat among the pigeons, but doesn't it open for the opportunity such that a company sets out its employer as candidate with mission to get GDAL money and fund the candidate time to develop a feature required by that company? Briefly, the company gets paid for what it needs.

Reply:
for substantial features, it will require PSC/supervisor approval, so this can limit a bit the risk of very niche features being funded through GDAL sponsorship dollars. And it remains to the appreciation of the PSC to select which candidates it wants, according to what he knows already about them.

Reply:
I guess this is possible, but the continuum of "needs" here is mostly going to be spread among maintenance, performance, and project improvement – not feature development. It isn't going to be a good vehicle for a company to get their favorite driver built and integrated. The PSC is the watch guard here.

That said, I hope there are (small) companies that step forward to use the vehicle to support themselves by doing work on GDAL. The idea of individual open source mercenaries looking for projects to contribute to is a myth just like unicorns. There are none of those to take this GDAL funding either (except for maybe @rouault 😄).

Reply:
@hobu Agreed. I just thought to spell an incidental thought out.